charts/eol-exporter/Chart.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v2
+name: eol-exporter
+description: |
+  End of life exporter.
+  A Kubernetes's helm chart for a exporter that get information about end of life/support of products in order to be scrapped by Prometheus
+
+  You must supply a valid configmap with a list of products with its versions. Check https://github.com/saritasa-nest/saritasa-devops-tools-eol-exporter/blob/main/config.yaml.example
+  for example values.
+  Each product must have a field `current` with valid version as defined in: https://endoflife.date/api/{product}.json
+  A `comment` field is optional, and it will be added as a label in the metrics.
+
+  A Prometheus extra scrape config must be configured in order to be able to watch the metrics in Prometheus.
+  The service name will be defined as: $CHART_NAME.$NAMESPACE:$PORT
+  By default this is: eol-exporter.prometheus:8080
+  An example extraScrapeConfigs is available in: https://github.com/saritasa-nest/saritasa-devops-tools-eol-exporter/blob/main/README.md#prometheus-server-config
+
+  The exporter provides two metrics:
+  - endoflife_expiration_timestamp_seconds: Information about end of life (EOL) of products. Metric value is the UNIX timestamp of the eolDate label
+  - endoflife_expired: Information about end of life (EOL) of products. Boolean value of 1 for expired products.
```

Comment:
provide a sample promql that we used in alerts as a quick ref

Reply:
Done